### PR TITLE
can now store data from multiple donations

### PR DESCRIPTION
**Pull request by cosmostevens**
- Updated the supportsBLM* portion of business.update() so that multiple sets of donation data can be added to their respective variables separated by " + "
- Added a charRemove to setSupportsBLMContribution so that there will not be extra double quotes anymore.
- Switched the business = busTemp to business.update(busTemp) so that supportBLM* data can be added on top of each other rather than lost

### CODE REVIEW

File: Website-and-Backend/src/main/java/com/example/appengine/springboot/Business.java [hunk 1]
```diff
@@ -3,84 +3,91 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
```
Comment (cosmostevens) [inline Website-and-Backend/src/main/java/com/example/appengine/springboot/Business.java:4]: I went through and alphabetized the Business class for better organization.

File: Website-and-Backend/src/main/java/com/example/appengine/springboot/Business.java [hunk 2]
```diff
@@ -3,84 +3,91 @@
...
-    boolean supportsBLM;
-    boolean blackOwnedBusiness;
+    String bcorpProfile;
+    String betterBusinessBureau;
+    String companyType;
+    String name;
+    String supportsBLMContribution;
+    String supportsBLMEntity;
+    String supportsBLMSource;
+    String website;
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/java/com/example/appengine/springboot/Business.java:28]: These should all be made private

Comment (cosmostevens) [reply Website-and-Backend/src/main/java/com/example/appengine/springboot/Business.java:28]: In c++ instance variables are automatically private if you put them before the public classes, but i guess they're not in java. My bad! I I'll change that up right now.

Comment (CubeOfCheese) [reply Website-and-Backend/src/main/java/com/example/appengine/springboot/Business.java]: By default variables are made "package" in java which means they are global within the scope of the package

**Pull request merged**